routes/web.php

```diff
@@ -24,7 +24,7 @@
  * Reports Endpoints
  */
 Route::get('/expense/report', 'ExpenseController@report')->name('expense.reports');
-Route::post('/subscribe', 'SubscriptionController@store');
+Route::post('/', 'SubscriptionController@store');
```

Comment:
Why this? You can't register a post request to the root URL 🤷‍♀️

Reply:
i keep getting error here on my pc with the test url i don't know if you get errors from your side

Reply:
but working fine locally